source/content/secure-runtime-access.md

```diff
@@ -17,6 +17,14 @@ In addition to defense in depth, this feature can be used to enforce role-based
 
 Secure Runtime Access is available to [contract](https://pantheon.io/plans/pricing) customers with an [Organization](/organizations) dashboard. [Contact Sales](https://pantheon.io/contact-us) to request that SRA be enabled for your site.
 
+### Considersations
+
+Users with Secure Runtime Access must have an active dashboard session to access database services. The following considerations apply:
```

Comment:
is logging in to the Pantheon dashboard via the browser the only way to start `an active dashboard session`?